ts/twitter/src/lib/components/setting-button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { IconButton, Tooltip } from "@chakra-ui/react";
+
+interface Props {
+  tooltipText: string;
+  ariaLabel: string;
+  icon: React.ReactElement;
+}
+
+export const SettingButton: React.FC<Props> = (props) => {
```

Comment:
Let's rename 'SettingButton' to 'SettingsButton'. Please make sure to also rename the file.

```suggestion
export const SettingsButton: React.FC<Props> = (props) => {
```